book-store/src/components/addBook.js

```diff
@@ -24,28 +25,30 @@ const AddBook = () => {
   };
 
```

Comment:
- [ ] [OPTIONAL] Good work on the project 👍🏻. I'd advise you to move the project directory to the default directory. By that, I mean to move all of the files inside of the `book-store` folder to outside of it. Creating a sub-directory to include the project, could lead to some issues with GitHub hosting in the future and is generally not recommended.